src/Pyz/Zed/OauthWarehouse/OauthWarehouseConfig.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+/**
+ * This file is part of the Spryker Commerce OS.
+ * For full license information, please view the LICENSE file that was distributed with this source code.
+ */
+
+namespace Pyz\Zed\OauthWarehouse;
+
+use Spryker\Zed\OauthWarehouse\OauthWarehouseConfig as SprykerOauthWarehouseConfig;
+
+class OauthWarehouseConfig extends SprykerOauthWarehouseConfig
+{
+    /**
+     * @var string
+     */
+    public const SCOPE_WAREHOUSE_USER = 'warehouse-user';
+
+    /**
+     * @var string
+     */
+    public const SCOPE_BACK_OFFICE_USER = 'back-office-user';
```

Comment:
- protected? are they used somewhere?
- looks useful to have `@uses` as in suite-nonsplit